jetty-server/src/main/java/org/eclipse/jetty/server/AsyncAttributes.java

```diff
@@ -0,0 +1,130 @@
+//
+//  ========================================================================
+//  Copyright (c) 1995-2020 Mort Bay Consulting Pty Ltd and others.
+//  ------------------------------------------------------------------------
+//  All rights reserved. This program and the accompanying materials
+//  are made available under the terms of the Eclipse Public License v1.0
+//  and Apache License v2.0 which accompanies this distribution.
+//
+//      The Eclipse Public License is available at
+//      http://www.eclipse.org/legal/epl-v10.html
+//
+//      The Apache License v2.0 is available at
+//      http://www.opensource.org/licenses/apache2.0.php
+//
+//  You may elect to redistribute this code under either of these licenses.
+//  ========================================================================
+//
+
+package org.eclipse.jetty.server;
+
+import java.util.HashSet;
+import java.util.Set;
+import javax.servlet.AsyncContext;
+
+import org.eclipse.jetty.util.Attributes;
+
+class AsyncAttributes extends Attributes.Wrapper
```

Comment:
Don't create a wrapper for this, just implement directly in `ServletAttributes`. See below: